config/webpack.config.js

```diff
@@ -34,7 +34,7 @@ const configGenerator = (options) => {
     filesToBuild = _.pick(entryFiles, options.entry.split(',').map(x => x.trim()));
   }
   filesToBuild.vendor = [
-    'core-js',
+    './src/js/common/polyfills',
```

Comment:
It includes this file and anything it depends on in the vendor bundle. It should not actually cause the require calls in the polyfills.js file to execute, though, which is why I left the `require('./polyfills')` statement in there. Similar to how React is in the vendor bundle, but it won't ever be executed unless you actually import/require it.

However, they're polyfills and it's possible some of them have code that is run before the actual `require` statement is called. I don't think that's the case, but it's possible.